.circleci/config.yml

```diff
@@ -24,6 +25,9 @@ jobs:
       - run:
           name: Run acceptance tests
           command: bundle exec rake features
+      - run:
```

Comment:
so far all the checks we are doing in danger seem to be a prerequisite for the build to start, maybe we could move this step before the `Build for release` step so we fail faster. If we ever have danger checks that need to access some output from the build such as code coverage we could have a pre build danger step and a post build danger step. What do you think?

Reply:
That's a good idea. We don't have unlimited CI so the earlier we fail the sooner we have CI resources available for more builds.